integration-test/src/main/java/org/commonjava/maven/ext/integrationtest/TestUtils.java

```diff
@@ -342,36 +368,32 @@ private static Integer runCommandAndWait( String command, String workingDir, Str
         String path = System.getenv( "PATH" );
         if (extraPath != null)
         {
-            path = extraPath + System.getProperty("path.separator") + path;
+            path = extraPath + File.pathSeparator + path;
         }
 
         Process proc = Runtime.getRuntime().exec(command, new String[] {"M2_HOME=" + MVN_LOCATION, "PATH=" + path}, new File(workingDir));
```

Comment:
I am not sure why it doesn't just specify the full path to the `mvn` command it wishes to use rather than try to set the `PATH` and run `mvn` without any path.

I also think there could be an issue where the `mvn` script actually comes from the system path and `M2_HOME` refers to the local maven copy.

Reply:
See comment above

Reply:
@dwalluck Does this need to be changed as well for consistency?

Reply:
There is also a reference to `M2_HOME` in `cli/src/main/java/org/commonjava/maven/ext/cli/Cli.java`.

The scripts have a references to `MAVEN_HOME` and I see no reference to `M2_HOME`, so I don't think this is doing anything at all and we could try removing it.

Reply:
Hmmm. On second thoughts, given I occasionally run on Maven 3.0.5 to check compatibility I think its better to leave it. ( I think it was removed >= 3.4 (?) )

Reply:
I checked and we use version 3.5.0 for the ITs, and this appears to be the first version where `M2_HOME` is gone from the scripts. For in the Java code, the property `${maven.home}` would be the right option for all versions since all it does is set `-Dmaven.home=$M2_HOME`.

The code we use for the settings isn't the same as upstream which also allows overriding the location, see https://github.com/apache/maven/blob/8d221f97fc72fbd9099e082600590fdf6b85143c/maven-core/src/main/java/org/apache/maven/settings/DefaultMavenSettingsBuilder.java#L68.

The latest versions have a new property, `maven.conf`. Old versions used `${maven.home}/conf`. All allow override via `MavenSettingsBuilder.ALT_GLOBAL_SETTINGS_XML_LOCATION = org.apache.maven.global-settings`.

Reply:
Ok nice investigation! Lets purge M2_HOME then.

Reply:
So when you said 3.0.5, did you mean 3.5.0? As you said, it will depend on the minimum Maven version requirement.

I still think we won't remove `M2_HOME`, but replace with `MAVEN_HOME` as the scripts do `-Dmaven.home=${MAVEN_HOME}`.

Reply:
But above the line `-Dmaven.home...` they do `MAVEN_HOME=`dirname "$PRG"`/..` ?